components/x-eventpromo/src/ImagesContainer.jsx

```diff
@@ -0,0 +1,16 @@
+import { h } from '@financial-times/x-engine';
+import Image from './Image';
+
+const ImagesContainer = ({eventpromoLink, image1, image2, image3}) => {
+
+	return (
+		<div className="event-promo-inarticle__blocks event-promo-inarticle__img-block">
+			<span className="event-promo__control js-event-promo__control" aria-label="control animation"></span>
```

Comment:
Should this be an interactive element such as a button?

Reply:
it's the play/pause button overlay-ed on top of the animated images, @Thurst-ft , any reason we should not make this a button?